packages/flame_rive/test/flame_rive_test.dart

```diff
@@ -105,6 +105,27 @@ void main() {
         );
       });
     });
+
+    group('Antialiasing', () {
+      test('Default value', () async {
+        final skillsArtboard = await loadArtboard(riveFile);
+        final riveComponent = RiveComponent(
+          artboard: skillsArtboard,
+        );
+
+        expect(riveComponent.artboard.antialiasing, isTrue);
+      });
+
+      test('Can change to false', () async {
+        final skillsArtboard = await loadArtboard(riveFile);
+        final riveComponent = RiveComponent(
+          artboard: skillsArtboard,
+          antialiasing: false,
```

Comment:
This parameter should be `antiAliasing` as it is two words.

Reply:
Instance variable is defined like that. If we change now it would be breaking change.

Reply:
Cant we deprecate it and make a getter to `antiAliasing`? That way it wont be breaking directly.

The constructor argument can work the same:

```
RiveComponent({
  @Deprecated('Use antiAliasing')
  bool? antialiasing,
  bool? antiAliasing,
}) : antiAliasing = antiAliasing ?? antialiasing ?? false;

bool antiAliasing;

@Deprecated('Use antiAliasing')
bool get antialiasing => antiAliasing;
set antialiasing(bool value) => antiAliasing = value;
```

Reply:
I think that we can do the breaking change directly, I don't think many were using this since it wasn't working.

Reply:
While it might not work, they might still have used it without knowing it. We shouldn't release breaking things on purpose imho

Reply:
None of the bridge libraries should have been post v1 to start with, but sure, we can do it non-breaking too.

Reply:
> Cant we deprecate it and make a getter to `antiAliasing`? That way it wont be breaking directly.
> 
> The constructor argument can work the same:
> 
> ```
> RiveComponent({
>   @Deprecated('Use antiAliasing')
>   bool? antialiasing,
>   bool? antiAliasing,
> }) : antiAliasing = antiAliasing ?? antialiasing ?? false;
> 
> bool antiAliasing;
> 
> @Deprecated('Use antiAliasing')
> bool get antialiasing => antiAliasing;
> set antialiasing(bool value) => antiAliasing = value;
> ```

I don't think it worth making a breaking change just for this type of spelling errors.

Reply:
Another way is to make change directly when there is big breaking change.

Reply:
That is why we deprecate it, they can still use it and ignore the deprecation warning if they so like.

Reply:
> That is why we deprecate it, they can still use it and ignore the deprecation warning if they so like.

but we have to remove it some time.

Reply:
And they can ignore it. But at least we have a migration away from it ahead of time. Not only are we making the code in-line with our style guides, we are also giving people the time to migrate away by warning them about it.

Reply:
Anyway, this PR only add the functionality of `antialiasing`. Can you file a new issue for name change?

Reply:
Since this PR didn't introduce the `antialiasing` argument I'd say we can go ahead and merge this.

Reply:
The [OG package](https://pub.dev/documentation/rive/latest/rive/Artboard/antialiasing.html) has the wrong name, since this is a bridge package, we should keep the mistakes from the OG one. IMO.